server/models/UserToken.ts

```diff
@@ -27,7 +28,7 @@ class UserToken extends Model<InferAttributes<UserToken>, InferCreationAttribute
   public declare lastUsedAt: CreationOptional<Date>;
 
   public declare user?: NonAttribute<User>;
-  public declare client?: NonAttribute<typeof models.Application>;
+  public declare client?: NonAttribute<OAuth2Server.Client>;
```

Comment:
This is incorrect, we're expecting an `Application` here

Reply:
I have made the changes.